process/interceptors/multiDataInterceptor.go

```diff
@@ -156,6 +165,10 @@ func (mdi *MultiDataInterceptor) ProcessReceivedMessage(message p2p.MessageP2P,
 		)
 	}
 
+	if !allWhiteListed && errOriginator != nil {
+		log.Debug("got message from peer on topic only for validators", "originator", message.Peer(), "topic", mdi.topic, "err", err)
```

Comment:
This should be changed to:
```GO
log.Debug("got message from peer on topic only for validators", 
"originator", p2p.PeerIdToShortString(message.Peer()), 
"topic", mdi.topic, 
"err", err)
``` 
as message.Peer() might contain unprintable characters
I think we should add a String() function on core.PeerID struct

Reply:
done